App/SB/components/RadioButton/index.js

```diff
@@ -4,14 +4,18 @@ import { View, TouchableOpacity, Image } from 'react-native'
 import styles from './statics/styles'
 
 const RadioButton = props => {
-  const { selected, style, disabled } = props
+  const { selected, style, disabled, onPress } = props
 
   return (
-    <TouchableOpacity style={[styles.button, style, disabled && styles.disabled]}>
+    <TouchableOpacity style={[styles.button, style, disabled && styles.disabled]} onPress={onPress}>
       { selected && <Image style={styles.buttonImage} source={require('./statics/check.png')} /> }
       { selected && <View style={[styles.buttonSelected, disabled && styles.buttonDisabled]} /> }
     </TouchableOpacity>
   )
 }
 
+RadioButton.defaultProps = {
```

Comment:
First, this component is just a simple stateless component. It's a function that takes in props and return a component. Setting `defaultProps` has no effect I think.

Reply:
Stateless components are no different from class components: they will pull in static properties like `defaultProps`/`propTypes` in the same way. I wasn't exactly sure where all else this radio button was being used, & had wanted to have a default in case!

Reply:
Wow cool, didn't know!